app/models/hackbot/helpers.rb

```diff
@@ -18,6 +18,12 @@ def action
       event[:action]
     end
 
+    # Returns whether the current event being processed is a PM. Slack channel
+    # IDs start with D if they're a private message.
+    def in_pm?
```

Comment:
Looks a bit repetitive with https://github.com/hackclub/api/pull/195/files#diff-fd1c65187e2901f60181c60c26649884R44.

How about we rename this to `in_pm?` and remove the method from `interactions/command.rb`?